packages/venia-concept/src/components/Gallery/gallery.js

```diff
@@ -13,21 +13,7 @@ class Gallery extends Component {
             pagination: string,
             root: string
         }),
-        data: arrayOf(
-            shape({
-                id: number.isRequired,
-                name: string.isRequired,
-                small_image: string.isRequired,
-                price: shape({
-                    regularPrice: shape({
-                        amount: shape({
-                            value: number.isRequired,
-                            currency: string.isRequired
-                        }).isRequired
-                    }).isRequired
-                }).isRequired
-            })
-        ),
+        data: array,
```

Comment:
In `RootComponents/Product/Product.js` we blew away the `data` propType entirely.

I'm not clear on the difference between these two, and when I would use each approach.

Reply:
Sorry, some comments got lost re-creating the PR, but the change in Product was just cleanup. The Product component doesn't receive its data as a prop; Jimmy and I had no idea why that existed at all. This component only receives `id` and `addToCart`.